packages/design-tokens/tokens/spacing.json

```diff
@@ -1,5 +1,53 @@
 {
   "spacing": {
+    "0": "var(--spacing-0, 0)",
+    "1": "var(--spacing-1, .0625rem)",
+    "2": "var(--spacing-2, .125rem)",
+    "4": "var(--spacing-4, .25rem)",
+    "6": "var(--spacing-6, .375rem)",
```

Comment:
Are these put here for our own internal use?

Reply:
Not sure, this file is autogenerated

Reply:
Apparently this file is created, then used to create the `less` and `scss` variables, but being `json` we can't write a comment to say this is auto-generated.

Reply:
Sorry, I meant to question their existence as a value we can use.

Reply:
Yeah I question their existence too. A card for the backlog? 😉

Reply:
I'm assuming this came from chatting with Elliot and what to do about values we use internally but aren't part of the original scale?

Reply:
I'd rather not expose what could potentially spawn a bunch of "off-scale" usage

Reply:
By off-scale usage @gyfchong do you mean adding the sizes that align to the older t-shirt equivalents?

Reply:
Not necessarily, if we're promoting a specific scale which consumers should use to space things apart (ie. the multiples of 4) then anything that doesn't conform to the scale is considered "off-scale". In this example, `1,2,6` are considered off scale. The scale is inherently there to make spacing math easier, so having these odd numbers might mess with that.

Although I just realised these are part of the Tailwind (new) spacing, which I thought hadn't included these so it didn't make sense to me as to why they'd be included in the design-tokens, so yeah we can take this discussion elsewhere.

Reply:
Ah gotcha, thanks for the clarification 👍